Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM eclipse-temurin:17.0.6_10-jre-alpine AS builder
+FROM --platform=${BUILDPLATFORM:-linux/amd64} eclipse-temurin:19-jre-jammy AS builder
```

Comment:
Do we need to upgrade to Java 19?

[17.0.6_10-jre-focal](https://hub.docker.com/layers/library/eclipse-temurin/17.0.6_10-jre-focal/images/sha256-b10df4660e02cf944260b13182e4815fc3e577ba510de7f4abccc797e93d9106?context=explore) looks like it's available

Reply:
<img width="1241" alt="image" src="https://user-images.githubusercontent.com/1526295/216288581-c529810e-5a5c-470f-a386-362f6114db65.png">

Reply:
@sldblog , I will try this version but all the teams that has adopted the multi arch build has upgraded to java19. That was the reason for me to use java 19

Reply:
Sure, but it feels like we should separate those steps, so if something goes wrong, we know if was from the switch to Ubuntu or from the switch to 19